### PR TITLE
support task level allocator

### PR DESCRIPTION
**Pull request by fengyun2066**
support task level allocator

### CODE REVIEW

**Comment by lizheng920625** [comment]
> 


/check-cla

**Comment by lizheng920625** [comment]
> Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
> The following commits have not yet signed CLA.
> 
> **[264bf65](https://github.com/kunpengcompute/boostkit-bigdata/commit/264bf6568dfb16ec41fb96f4120611f9974305b0)** | support task level allocator
> 
> 📝 **Please access [here](https://clasign.osinfra.cn/sign/Z2l0aHViJTJGa3VucGVuZ2NvbXB1dGU=) to sign the CLA.**
> 
> It may take a couple minutes for the CLA signature to be fully registered; after that, please reply here with a new comment: **/check-cla** to verify. Thanks.
> 
> * If you've already signed a CLA, it's possible you're using a different email address for your gitee account. Check your existing CLA data and verify the email.
> * If you signed the CLA as an employee or a member of an organization, please contact your corporation or organization to verify you have been activated to start contributing.
> * If you have done the above and are still having issues with the CLA being reported as unsigned, please feel free to file an issue.

/check-cla

**Comment by tushengxia** [comment]
>

File: omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengVecAllocator.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
+package nova.hetu.olk.memory;
```
Comment (tushengxia) [inline omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengVecAllocator.java:1]: 请补充license和copyright

File: omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengAllocatorFactory.java [hunk 2]
```diff
@@ -0,0 +1,88 @@
+package nova.hetu.olk.memory;
```
Comment (tushengxia) [inline omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengAllocatorFactory.java:1]: 请补充license和copyright

File: omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengVecAllocator.java (hunk 1 above)
Comment (fengyun2066) [reply omnioperator/omniop-openlookeng-extension/src/main/java/nova/hetu/olk/memory/OpenLooKengVecAllocator.java:16]: done

**Pull request merged**